### PR TITLE
GDB-7891: add icon to "Download as" dropdown.

### PR DESCRIPTION
**Pull request by boyan-tonchev**
## What
Adds icon "download" to "Download as" dropdown. When screen is smaller than 768px only the icon will be visible. 

## Why
This will improve UX, and is consistent with plugin tabs names.

## How
"Dropdown" component hase improved to support icon. Changed the setup of "Dropdown" component in "Download as" with download icon.

### CODE REVIEW

File: ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx [hunk 1]
```diff
@@ -30,11 +31,12 @@ export class Dropdown {
   render() {
     return (
       <div class="ontotext-dropdown">
-        <button class={`ontotext-dropdown-button ${this.open ? 'icon-caret-up-after' : ' icon-caret-down-after'}`}
-                onClick={() => this.toggleComponent()}>
-          {this.translationService.translate(this.nameLabelKey)}
-        </button>
-
+        <div class="ontotext-dropdown-button-wrapper" onClick={() => this.toggleComponent()}>
```
Comment (svilenvelikov) [inline ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx:34]: We need a tooltip when this shrinks

Comment (svilenvelikov) [inline ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx:34]: Using non actionable elements like a `div` with a clickhandler is bad from UX point of view. Better, move the icon inside the button and return the clickhandler on it.

**Review by svilenvelikov — CHANGES_REQUESTED** [review]

File: ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx (hunk 1 above)
Comment (boyan-tonchev) [reply ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx:34]: I had problem with styling when icon is inside the button but I will try again.

Comment (boyan-tonchev) [reply ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx:34]: This is a good idea.

File: ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.tsx [hunk 2]
```diff
@@ -293,6 +293,14 @@ export class OntotextYasguiWebComponent {
     });
   }
 
+  @Listen('resize', { target: 'window' })
+  handleScroll() {
```
Comment (svilenvelikov) [inline ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.tsx:297]: Rename the method! I guess this is copied from somewhere maybe

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**